components/event-bus/internal/knative/util/kn-eventing.go

```diff
@@ -159,9 +159,9 @@ func (k *KnativeLib) GetChannelByLabels(namespace string, labels *map[string]str
 }
 
 // CreateChannel creates a Knative/Eventing channel controlled by the specified provisioner
-func (k *KnativeLib) CreateChannel(name string, namespace string, labels *map[string]string,
+func (k *KnativeLib) CreateChannel(prefix string, namespace string, labels *map[string]string,
```

Comment:
maps are pointers, so `*map` is a big no-no.
Redundant `string`.
```suggestion
func (k *KnativeLib) CreateChannel(prefix, namespace string, labels map[string]string,
```